docs/acme-divergences.md

```diff
@@ -50,66 +54,73 @@ from the directory instead.
 
 Boulder does not provide the "index" link relation pointing at the directory URL.
 
-## [Section 7.1.2](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.1.2)
+## [Section 7.1.2](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.1.2)
 
-Boulder does not implement the `terms-of-service-agreed` or `orders` fields in the registration object (nor the endpoints the latter links to).
+Boulder does not implement the `termsOfServiceAgreed` or `orders` fields in the registration object (nor the endpoints the latter links to).
 
-## [Section 7.1.3](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.1.3)
+## [Section 7.1.3](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.1.3)
 
 Boulder does not implement orders (previously called `applications`), instead it implements the `new-cert` flow from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5). Instead of authorizations in the order response, Boulder currently uses authorizations that are created using the `new-authz` flow from [draft-ietf-acme-02 Section 6.4](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.4).
 
-## [Section 7.1.4](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.1.4)
+Any identifier of type "dns" in a new-order request MAY have a wildcard domain name as its value, which is not supported by Boulder.
+
+## [Section 7.1.4](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.1.4)
 
 Boulder does not implement the `scope` field in authorization objects.
 
-## [Section 7.2](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.2)
+## [Section 7.2](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.2)
 
 Boulder doesn't implement the `new-nonce` endpoint, instead it responds to `HEAD` requests with a valid `Replay-Nonce` header per [draft-ietf-acme-03 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.4).
```

Comment:
This should be updated to say "Boulder doesn't implement the `NewNonce` endpoint".